edn/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Generated by Cargo
+# will have compiled files and executables
+/target/
```

Comment:
I think this file shouldn't be needed - our toplevel gitignore should handle these paths (or be updated to handle them)